srcpkgs/ffmpeg/template

```diff
@@ -24,6 +24,7 @@ makedepends="zlib-devel bzip2-devel freetype-devel alsa-lib-devel libXfixes-deve
  $(vopt_if nvenc nv-codec-headers) $(vopt_if sndio sndio-devel)
  $(vopt_if dav1d libdav1d-devel) $(vopt_if zimg zimg-devel)
  $(vopt_if webp libwebp-devel) $(vopt_if sofa libmysofa-devel)"
+depends="ffplay"
```

Comment:
So make it possible to `ignorepkg` it, basically? And I think this should use `ffplay>=${version}_${revision}`